src/views/partials/breaches.js

```diff
@@ -31,11 +31,17 @@ function createBreachRows (data) {
         addedDate: longDate.format(addedDate),
         dataClasses: longList.format(dataClassesTranslated)
       })
+      const logoPath = logos.get(breach.Domain) ?? '/images/fallback-logo.svg'
 
       return `
       <details class='breach-row' data-status=${status} data-email=${account.email} ${isHidden ? 'hidden' : ''}>
         <summary>
-          <span>${breach.Title}</span>
+          <span>
+            <a href='https://${breach.Domain}' target='_blank' class='breach-company'>
```

Comment:
nit: maybe a `rel="noopener noreferrer nofollow"` on this link (if it needs to be linked) since some of these sites are sometimes a bit sketchy.
I've never been convinced the domain needs to be linked. I think last time I checked, it failed like 40% of the time if the site was no longer a site, or wasn't https:// compatible.

Reply:
Also, does this need to do a check for `breach.Domain`? I think all records have them, but if I do a quick Ctrl+F on `Domain: ""` I get about 26 results (unless those results have already been filtered out). Although I do see a nullish check on `breach.Domain` above on L34, so&hellip; /suspicious_eyes

Reply:
Well, now I'm getting conflicting input :sweat_smile: https://github.com/mozilla/blurts-server/pull/2825#discussion_r1105046801

But hopefully PM agrees that the link can go away and it won't be an issue. If not, then yes, this probably needs such a check, good callout.

Reply:
Y'all gonna make me cite sources tonight eh? 😄 

Regarding `rel="noreferrer"`:
>It indicates that no referrer information is to be leaked when following the link _**and also implies the `noopener` keyword behavior under the same conditions**_.
https://html.spec.whatwg.org/multipage/links.html#link-type-noreferrer

>...and creates a top-level browsing context as if `noopener` were also set
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel#attr-noreferrer

>Sick of typing `<a rel="noopener noreferrer">`?
`<a rel="noreferrer">` implies noopener.
Specifying both is redundant 🤪
https://twitter.com/_developit/status/1054422343609655296

Regarding [our CSP](https://github.com/mozilla/blurts-server/blob/a27170725ff114f5859a4bedc2efd72c4ed93b34/src/app.js#L119-L123) which is actually stricter than `no-referrer`! (`no-referrer` is our fallback):
>Using Referrer-Policy: no-referrer is effectively setting `rel=noreferrer` on every link and protects the most against information leakage.
https://kevinhakanson.com/2018-02-07-referrer-policy-http-header

Regarding `rel="nofollow"`:
This seems more SEO-related but probably worthy of inclusion, particularly for links like these!

Reply:
Interesting, I didn't know this: https://mathiasbynens.github.io/rel-noopener/

> _"2021 update: Browsers now implicitly set `rel=noopener` for any `target=_blank` link, [following a spec change](https://github.com/whatwg/html/issues/4078). If the demo on this page no longer seems scary, congratulations — you’re using a modern browser!"_

TL:DR: Remove the link and lets never discuss this again. /disappears into fog

Reply:
Links removed in c56c084b.